src/app/Processor/ProcessorDetailPage/ProcessorDetailPage.tsx

```diff
@@ -121,33 +216,64 @@ const ProcessorDetailPage = (): JSX.Element => {
                 <Stack hasGutter={true}>
                   <StackItem>
                     <TextContent>
-                      <Text component="h1">{processor.name}</Text>
+                      <Text component="h1">{currentProcessor.name}</Text>
                     </TextContent>
                   </StackItem>
                   <StackItem>
-                    <StatusLabel status={processor.status ?? ""} />
+                    <StatusLabel status={currentProcessor.status ?? ""} />
                   </StackItem>
                 </Stack>
               </SplitItem>
-              <SplitItem>
-                <Dropdown
-                  onSelect={actionsSelect}
-                  toggle={
-                    <DropdownToggle
-                      id="toggle-id"
-                      onToggle={actionsToggle}
-                      toggleIndicator={CaretDownIcon}
-                    >
-                      {t("common.actions")}
-                    </DropdownToggle>
-                  }
-                  isOpen={isActionsOpen}
-                  dropdownItems={actionItems}
-                />
-              </SplitItem>
+              {!isEditing && (
+                <SplitItem>
+                  <Split hasGutter={true}>
+                    <SplitItem>
+                      <Button
+                        isAriaDisabled={
+                          currentProcessor.status !==
```

Comment:
@vpellegrino Is it not possible to edit processor twice in the mock server? After the editation, the state is accepted and it is not changed to ready.

Reply:
The state should transition to "ready" after few seconds. Then, it should be possible to edit the processor again

Reply:
@vpellegrino Thank you, I tried to refresh InstancePage so I lost data. Currently I go back through the breadcrumb item and it works.